_dashboards/index.md

```diff
@@ -11,15 +11,42 @@ redirect_from:
+OpenSearch Dashboards is an integrated visualization tool that makes it easy for users to explore their data in OpenSearch. From real-time application monitoring, threat detection, and incident management to personalized search, OpenSearch Dashboards gives you data visualization tools to tell your data story.
 
+Out-of-the-box OpenSearch Dashboards includes several tools for discovering and visualizing data, and the OpenSearch suite provides [plugins](https://opensearch.org/docs/latest/dashboards/install/plugins/) to extend and customize dashboard features and functions. 
 
-## Get started with OpenSearch Dashboards
+# Use cases for real-time search and log analytics at scale 
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+ OpenSearch is powered by the Apache Lucene search library, and it supports a number of search and analytics capabilities, such as k-nearest neighbors (k-NN) search, SQL, anomaly detection, trace analytics, full-text search, and more.  
+
+<table bgcolor="lightgrey" width="600">
+<tbody>
+<tr>
+<td style="text-align: center; vertical-align: top; font-weight: bold;">Visualize log and trace data with interactive log analytics.<img src="{{site.url}}{{site.baseurl}}/images/visualize-log-data.png" alt="Log analytics"/></td>
+<td style="text-align: center; vertical-align: top; font-weight: bold;">Detect and mitigate issues faster with anomaly detection.<img src="{{site.url}}{{site.baseurl}}/images/anomaly-detection.png" alt="Anomaly detection"/></td>
+</tr>
+<tr>
+<td style="text-align: center; vertical-align: top; font-weight: bold;">Diagnose performance issues and reduce application downtime using the observability interface and log monitoring features.<img src="{{site.url}}{{site.baseurl}}/images/observability.png" alt="Observability interface and log monitoring features" /></td>
+<td style="text-align: center; vertical-align: top; font-weight: bold;">Unlock real-time search, monitoring, and analysis of business and operational data.<img src="{{site.url}}{{site.baseurl}}/images/analyzing-data-logs.png" alt="Real-time search, monitoring, and analysis" /></td>
+</tr>
+</tbody>
+</table> 
+
+# Explore the OpenSearch Dashboards playground
+
+Follow these steps to explore the basics of OpenSearch Dashboards:
+
+1.  Go to the [OpenSearch Dashboards playground](https://playground.opensearch.org/app/home).
+2.  Choose **OpenSearch Dashboards > Dashboards > Create dashboard**. 
+3.  Explore the sample Global Flight, Web Traffic, and Revenue dashboards. 
+
+# Install OpenSearch Dashboards
```

Comment:
In the previous iteration, we had two ways of trying out dashboards:
1. In the playground
2. After installing locally.
Please give the users both options. The steps are essentially the same except if you're doing it locally, you have to install dashboards first and then add data for the dashboard you want to view (flight, e-commerce, etc.).

Reply:
The install instructions will give this option. The main point of this Gettting started is to let them try it out before going to the install steps, which is a separate PR and gives users the install locally option. The install PR is in tech review.

Reply:
It seems that the install PR will show the users how to install dashboards. Does it also guide them how to try out dashboards?

Reply:
Yes